azerothcore/azerothcore-wotlk
# fix(DB/pool): Pool herbs in Alterac Mountains

## Changes Proposed:
-  Pools the previously-unpooled Alterac Mountains

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/6778
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7384
- Closes https://github.com/chromiecraft/chromiecraft/issues/1144
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/6254
- Closes https://github.com/chromiecraft/chromiecraft/issues/794

## SOURCE:

Vmangos

## Tests Performed:

- Not tested yet.
- 


## How to Test the Changes:


1. Preferably download a Gatherer/Gathermate addon
2. Fly around the zone picking all herbs (use `.cheat casttime` `.add 23162 4`)
3. See if the quantities found are correct/rational
4. See if they respawn after a while

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
